source/LibKuribo/modules/kxer/Loader.cxx

```diff
@@ -287,6 +282,15 @@ LoadResult Load(const LoadParam& param, LoadedKXE& out) {
   if (result != LoadResult::Success)
     return result;
 
+#if KURIBO_PLATFORM_WII
+  // Cache blocks are 32 bytes in size
+  for (int i = 0; i < pHeader->code.file_size; i += 32) {
```

Comment:
Signed-unsigned comparison here. Additionally, we should assert pCode is 32byte aligned here. Otherwise LGTM.

Reply:
Actually, we don't require pCode to have any particular alignment--that's configurable. We should really just adapt this loop to work on non-aligned segments.

Reply:
Changed it. All that had to be done was align the pointer down to 32 bytes since according to specs it will still refresh all necessary cache blocks

Reply:
This isn't quite right. Imagine the code was 32B spanning two cache blocks. Here we'd only invalidate the first one.

Reply:
This still isn't quite right since fileSize may not be 32B aligned either. I'd just keep two iterators, `cacheBegin` set to the low address rounded down and `cacheEnd` set to the high address rounded up. The loop then would be of the form `for (auto* it = cacheBegin; it < cacheEnd; it += 32)`.